ha/setup/ha_setup.py

```diff
@@ -151,6 +151,8 @@ def process(self):
             PostInstallCmd.remove_file(const.CM_CONTROLLER_SCHEMA)
             shutil.copyfile(f"{const.SOURCE_CONFIG_PATH}/{const.CM_CONTROLLER_INDEX}.json",
                             const.CM_CONTROLLER_SCHEMA)
+            PostInstallCmd.remove_file(const.ALERT_FILTER_RULES_FILE)
+            shutil.copyfile(const.SOURCE_ALERT_FILTER_RULES_FILE, const.ALERT_FILTER_RULES_FILE)
```

Comment:
We want to delete this file in the "cleanup" command, please add that.

Reply:
Sure

Reply:
Done in the latest patch